Doppler.AccountPlans/Model/Promotion.cs

```diff
@@ -0,0 +1,10 @@
+namespace Doppler.AccountPlans.Model
+{
+    public class Promotion
+    {
+        public int? IdPromotion { get; set; }
+        public int? IdUserTypePlan { get; set; }
+        public int? ExtraCredits { get; set; }
+        public int? DiscountPlanFee { get; set; }
```

Comment:
Are all of these properties really nullable? or at least some of them will always have a value?

Reply:
IdPromotion and IdUserTypePlan  should not be nullable, ExtraCredits DiscountPlanFee yes because these values depends on the promocode type, I'll do

Reply:
I talked with @jhoffman-ms, and we remove the properties IdPromotion and IdUserTypePlan  because are not necessary to expose these values in frontend

Reply:
❤️